lib/elastic_whenever/task/rule.rb

```diff
@@ -53,19 +63,22 @@ def create
       end
 
       def delete
+        Logger.instance.message("Listing Targets by Rule: #{name}")
         targets = client.list_targets_by_rule(rule: name).targets
+        Logger.instance.message("Removing Targets") unless targets.empty?
         client.remove_targets(rule: name, ids: targets.map(&:id)) unless targets.empty?
+        Logger.instance.message("Removing Rule: #{name}")
```

Comment:
From the user's perspective, the log about removing targets and listing targets may be redundant. The biggest concern is which rule was deleted, and the rule and target are now one-to-one, so it may be enough to just output the rule.